account_invoice_validate_send_email/models/mail_compose_message.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MailComposeMessage(models.TransientModel):
+    _inherit = "mail.compose.message"
+
+    @api.multi
+    def get_mail_values(self, res_ids):
+        self.ensure_one()
+        results = super(MailComposeMessage, self).get_mail_values(res_ids)
+        reply_to = self.reply_to
+        for res_id in res_ids:
+            results[res_id]["reply_to"] = reply_to
```

Comment:
```suggestion
        for res_id in res_ids:
            if self.reply_to and not results[res_id].get("reply_to", True):
                results[res_id]["reply_to"] = self.reply_to
```
Wasn't perfectly sure if self.reply_to would be perfectly reliable (does it always keep the value from the template?), but I guess we can go with this for now.

Let's add the condition `if self.reply_to and not results[res_id].get("reply_to", True):` to limit the scope of this change. There can be cases where `self.reply_to` is False and in that case it could mess up the outcome if we set False to `results[res_id]["reply_to"]` - the email from the system parameters will not be applied.

Reply:
I understood there is a case that `self.reply_to` is False. Now, I look back my work and thought until yesterday, I faced the case repeatedly because I failed to recognize requirement of template settings of reply_to.
I'll change the codes soon.